app/types/BalanceStatus.d.ts

```diff
@@ -1,5 +1,6 @@
 import type { StringUInt64 } from './SpecialStrings';
 
+// TODO rename this so it's no longer Pmob. not correct term when referring to eUSD
```

Comment:
Suggestion: Pull out TODO into a github issue, or pivotal/asana task for tracking/visibility

Reply:
I'll add it to the pivotal board